src/pybind/cephfs/cephfs.pyx

```diff
@@ -768,7 +768,7 @@ cdef class LibCephFS(object):
         with nogil:
             ret = ceph_mkdir(self.cluster, _path, _mode)
         if ret < 0:
-            raise make_ex(ret, "error in mkdir '%s'" % path)
+            raise make_ex(ret, "error in mkdir {}".format(path.decode('utf-8')))
```

Comment:
I think it would be better here to use `b"error in mkdir {}..."` I would prefer we do not use unicode in the cephfs binding.

Reply:
Okay, I will remove this patch.